client/src/Components/UserInformation.js

```diff
@@ -33,104 +54,111 @@ const InterviewLevelInfo = ({ interviewLevel }) => {
           Used to job interviews and looking for a challenge
         </div>
       </>
-    );
+    )
   }
-};
+}
 
 const UserInformation = (props) => {
-  const classes = useStyles();
-  const { dispatch } = useContext(store);
-  const history = useHistory();
+  const classes = useStyles()
+  const { dispatch } = useContext(store)
+  const history = useHistory()
 
   const [userData, setUserData] = useState({
-    language: 'English',
+    language: "English",
     experience: 0,
     interviewLevel: 1,
-  });
+  })
 
   const changeRating = (e) => {
-    console.log(e.target.value);
-    setUserData({ ...userData, [e.target.name]: e.target.value });
-  };
+    setUserData({ ...userData, interviewLevel: Number(e.target.value) })
+  }
+
+  const changeExperience = (e) => {
+    setUserData({ ...userData, experience: e.target.value })
+  }
+
+  const changeLanguage = (e) => {
+    setUserData({ ...userData, language: e.target.value })
+  }
 
   const onSubmit = async (e) => {
-    e.preventDefault();
+    e.preventDefault()
 
-    let formData = { ...props.formData, ...userData };
-    console.log(formData);
+    const id = props.user._id
 
     try {
-      const result = await axios.post(
-        'http://localhost:3001/users/register',
-        formData
-      );
-      dispatch({ type: USER_LOADED, payload: result.data.user });
-
-      const token = result.data.token;
-      localStorage.setItem(process.env.REACT_APP_USER_DATA, token);
+      const result = await axios.put(
+        `http://localhost:3001/users/update/${id}`,
+        userData
+      )
+      dispatch({ type: USER_LOADED, payload: result.data.user })
+
       // will change to /background (protected route, routes folder)
-      history.push('/dashboard');
+      history.push("/dashboard")
     } catch (error) {
-      console.log(error);
+      console.log(error)
     }
-  };
+  }
 
-  const { interviewLevel } = userData;
+  const { language, experience, interviewLevel } = userData
 
   return (
-    <>
-      <div className={classes.getStarted}>
-        <h1>Tell us about your Backgroud</h1>
+    <div className={classes.backgroundContainer}>
+      <GlobalCss />
+      <div className={classes.background}>
+        <h1 className={classes.backgroundHeader}>Tell us about your Background</h1>
+
+        <form className={classes.backgroundForm} id="backgroundForm">
+          <div className={classes.infoFormDiv}>Your Language:</div>
+          <Select
+            className={classes.infoDropdown}
+            onChange={changeLanguage}
+            value={language}
+          >
+            <MenuItem value="English">English</MenuItem>
+          </Select>
+
+          <div className={classes.infoFormDiv}>
+            Years of professional experience:
+          </div>
+          <Select
+            className={classes.infoDropdown}
+            onChange={changeExperience}
+            value={experience}
+          >
+            <MenuItem value={0}>0</MenuItem>
```

Comment:
Tiny thing but you can use Array.from() to make a list of MenuItems and make it more readable
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from?v=example#Browser_compatibility